mlir/test/Gradient/ClassicalJacobianTest.mlir

```diff
@@ -66,59 +62,61 @@ func.func @gradCall(%arg0: tensor<3xf64>) -> tensor<3xf64> {
 
 // -----
 
-// CHECK-LABEL: @structured_circuit.argmap(%arg0: f64, %arg1: i1, %arg2: i1) ->  tensor<?xf64>
-func.func @structured_circuit(%arg0: f64, %arg1: i1, %arg2: i1) -> f64 attributes {qnode, diff_method = "parameter-shift"} {
+// CHECK-LABEL: @structured_circuit.argmap(%arg0: tensor<1xf64>, %arg1: i1, %arg2: i1, %arg3: memref<?xf64>)
+func.func @structured_circuit(%arg0: tensor<1xf64>, %arg1: i1, %arg2: i1) -> f64 attributes {qnode, diff_method = "parameter-shift"} {
```

Comment:
In this file as well I'm just curious whether the changes (wrapping the scalars in tensors) were incidental or for a specific purpose (e.g. to diversify the tests)?

Reply:
I think this was my doing because there was some scalar instead of tensors and before Jacob's PR we where not supporting it